validator/schemas/entity-schema-v1.json

```
@@ -211,6 +211,10 @@
     "dashboardTemplates": {
       "$id": "#/properties/dashboardTemplates",
       "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^([aA-zZ_-])*(dashboard.json)$"
```

```suggestion
        "pattern": "^([a-zA-Z_-])*(dashboard.json)$"
```

Is this what you where looking for? 

The current regex will match [, \, ], ^, _, and `

---

Also supper nit but the regex doesn't need the capture groups `^[aA-zZ_-]*dashboard.json$`

---

Thanks! you're right!